web-module/src/main/webapp/templates/celAjax/redirectToNextDocRef.vm

```diff
@@ -0,0 +1,33 @@
+#set($pageType = $request.get('pageType'))
+#if($services.pageType.getPageTypeConfig("$!pageType"))
+  #set($template = $request.get('template'))
+  #if("$!template" == '')
+    #set($template = "celements2web:Templates.$pageType")
+  #end
+  #set($namingMode = $request.get('namingMode'))
+  #set($prefix = $request.get('prefix'))
+  #set($spaceRef = $services.reference.create().space($pageType).build())
```

Comment:
The space may not be influencable by the user. This has security implications. The system must only accept users of a restricted space (so far this is the XWiki-space). We further should think about expecting "register" rights for creating documents in the XWiki space. Yet, this is a topic of an other issue.
Please change the logic, that the space cannot be influenced by request-parameters.